notebook/notebookapp.py

```diff
@@ -187,6 +187,7 @@ def init_settings(self, ipython_app, kernel_manager, contents_manager,
             },
             version_hash=version_hash,
             ignore_minified_js=ipython_app.ignore_minified_js,
+            nbextensions=ipython_app.nbextensions,
```

Comment:
Again, I want to keep it simple. I think that allowing updating this while the server runs is out of scope. But, there is nothing to prevent someone else from just calling `utils.load_extensions()` in the frontend at any time to load an extension. We could even provide an IPython magic that does that, or another extension could be written to handle runtime loading.